lib/alert/addon/project/route.js

```diff
@@ -0,0 +1,40 @@
+import { next } from '@ember/runloop';
```

Comment:
Same as with cluster, if you can mount inside the regular project route that would be good, if not import and reexport it instead of duplicating all the code.

Reply:
Cluster and project are not the same! The duplicated part is small & it makes the condition less complicated. But I'll the way the Alert engine is mounted. But there is an issue I still can't figure out.